client/data/settings/actions.js

```diff
@@ -43,24 +43,18 @@ export function updateSettings( data ) {
 }
 
 export function updateIsWCPayEnabled( isEnabled ) {
-	return {
-		type: ACTION_TYPES.SET_IS_WCPAY_ENABLED,
-		isEnabled,
-	};
+	// eslint-disable-next-line camelcase
+	return setSettingsValue( { is_wcpay_enabled: isEnabled } );
```

Comment:
How do we decide when to use an object vs individual properties inside the object as the action? I'm guessing one trade off here is if we want re-rendering for the entire object vs just the individual property? Are there more things to considered or just for the simplicity of code structure?